Music/pianobarctl.15s.sh

```diff
@@ -109,8 +109,8 @@ fi
 echo pianobar #$state
 
 echo ---
-[ $state = "on" ] && echo $nowplaying 
-[ $state = "off" ] && echo "Launch | bash='$0' param1=launch terminal=false refresh=true"
-[ $state = "on" ] && echo "Pause / Unpause | bash='$0' param1=togglepause terminal=false"
-[ $state = "on" ] && echo "Next | bash='$0' param1=next terminal=false"
-[ $state = "on" ] && echo "Quit | bash='$0' param1=quit terminal=false refresh=true"
+[ $state = "on" ] && echo "$nowplaying" 
+[ $state = "off" ] && echo "Launch | bash="$0" param1=launch terminal=false refresh=true"
```

Comment:
I don't think this is going to do what you want.   By making those unescaped double quotes, you have no quotes areound `$0` on output.  You are probably wanting `bash=\"$0\"`

Without the quotes, this will lead to a problem if someone has a space in their path to the bitbar plugin as it would output as:
```
bash=/path/to/bitbar plugin/pianobarctl.15s.sh
```
where `plugin/pianobarctl.15.sh` would be an invalid option by bitbar